pkg/shared/utils/utils.go

```diff
@@ -71,6 +71,20 @@ func Int32PointerValOrEmpty(val *int32) int32 {
 	return *val
 }
 
+func Float32PointerValOrEmpty(val *float32) float32 {
```

Comment:
I'd prefer to use generic here if possible. Something like this: https://go.dev/play/p/6p596MSAHhD?v=goprev